backend/database_handler/transactions_processor.py

```diff
@@ -42,6 +43,15 @@ def _parse_transaction_data(transaction_data: Transactions) -> dict:
             ],
         }
 
+    @staticmethod
+    def _transaction_data_to_str(data: dict) -> str:
```

Comment:
Awesome! Could you add some function docstring explaining a bit why are we serializing bytes as base64?

Reply:
Because json.dumps doesn't support bytes?...

Reply:
didn't know that! I think it's not common knowledge, so I'd love to have a comment there mentioning that :)